README.md

```diff
@@ -39,40 +41,58 @@ settings] you may want:
     </script>
     ```
 
-## Required setting
+## Configuration
 
-There is only one - your public API key. You can get that by creating an
-account [Uploadcare]. You can use demo public key during dev stage, but note that
-demo account files are removed every few hours.
+### Required settings
 
+In fact, there's only one required setting — your public API key. In order to get the key
+you should register with Uploadcare and set up a project to store your files and settings.
+Check out our [knowledgebase][kb_intro] to get up an running with your Uploadcare account
+in minutes. You can always use our demo public key during your dev stage. But keep in mind,
+files uploaded to the demo account are removed every few hours.
```

Comment:
Like, isn't it great to know you can always try something out without signing up? If they've got files of their own, it won't feel comfy to upload them to some public demo account anyway. U know wat I'm sayin?